//test error

void foo() { return; }

int main() {

  foo(true);  
  return 3;

}